service/worker/scheduler/testdata/generate_history.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -x
+
+id=sched1
+
+trap "temporal schedule delete -s $id" EXIT
+
+temporal schedule create -s $id \
+  --overlap-policy bufferall \
+  --interval 10s \
```

Comment:
Why not a shorter interval to make this run faster?

Reply:
it's doing some odd sleeps below and I didn't want the timing to be so tight. it only has to be run manually once in a while so it's not a big deal.